README.md

```diff
@@ -13,5 +13,9 @@ yarn install
 ## Usage
 
 ```bash
+node localScrapper.js # Prepares karakuri-ready data from karaoke files
 npm start
-```
\ No newline at end of file
+```
+
+## Options
+Run `node localScrapper.js --help` and `node start.js --help` to get a list of options.
```

Comment:
You're right, but I hate the -- to get arguments to script used in npm scripts :( (I'll do the changes, but with a tiny amount of reluctance I feet compelled to mention =D)